eng/source-build-release.yml

```diff
@@ -0,0 +1,121 @@
+trigger: none
+pr: none
+
+pool:
+  name: NetCore1ESPool-Svc-Internal
+  demands: ImageOverride -equals 1es-ubuntu-2004
+
+parameters:
+  - name: sdkVersion
+    displayName: SDK Version
+    type: string
+    default: 6.0.XYY
+  - name: useCustomTag
```

Comment:
From a usability perspective, I see value in having only one parameter around the custom tag.  If customTag is non-null use it.  This avoids having to specify multiple values.

Reply:
That's what I wanted, but there is no way to have a null input in Azure pipelines. There are no optional parameters and you cannot start the pipeline with an empty input.

Reply:
Can't you use an empty string though?  Don't define a default value.

Reply:
I've tried. If you use an empty string, or do not specify a default value, then you are not allowed to start the pipeline until you input something. Also see https://developercommunity.visualstudio.com/t/optional-runtime-parameter/975396

Reply:
Interesting, I did not know this.  This is rather limiting.  If you define parameters in the AzDO build definition itself instead of the yaml you can utilize the empty string trick.